From 6bddf96edc0b3aa77cc900b3992e1117b90eb768 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 19 Apr 2016 09:02:46 +0800
Subject: [PATCH 1017/5242] MLK-12676-1 arm: dts: enable usb otg for imx6ull
 arm2 board

commit  c05728082ba820239456d6b0bad85c323b8dd91a from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable otg mode for OTG1 port for imx6ull arm2 board.

Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index 1c69d8b..f861eb9 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -913,9 +913,12 @@
 };
 
 &usbotg1 {
+	vbus-supply = <&reg_usb_otg1_vbus>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usb_otg1_id>;
-	dr_mode = "peripheral";
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.7.9.5

